trino-s3-proxy/src/main/java/io/trino/s3/proxy/server/rest/ParsedS3Request.java

```diff
@@ -33,6 +33,7 @@ public record ParsedS3Request(
         MultivaluedMap<String, String> requestHeaders,
         MultivaluedMap<String, String> requestQueryParameters,
         String httpVerb,
+        Optional<String> rawQuery,
```

Comment:
Should we also add the region as part of the parsed s3 request?

Reply:
Sure why not. Let me see if it's easy to add.

Reply:
It's not simple from what I can see. Maybe you can do it in a separate Pr?